sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/query.sql

```diff
@@ -0,0 +1,643 @@
+      HAVING
+        MIN submission_timestamp
+    ) AS distribution_id,
+    ANY_VALUE(
+      environment.partner.distribution_version
+      HAVING
+        MIN submission_timestamp
+    ) AS partner_distribution_version,
+    ANY_VALUE(
+      environment.partner.distributor
+      HAVING
+        MIN submission_timestamp
+    ) AS partner_distributor,
+    ANY_VALUE(
+      environment.partner.distributor_channel
+      HAVING
+        MIN submission_timestamp
+    ) AS partner_distributor_channel,
+    ANY_VALUE(environment.partner.partner_id HAVING MIN submission_timestamp) AS partner_id,
+    ANY_VALUE(
+      environment.settings.attribution.campaign
+      HAVING
+        MIN submission_timestamp
+    ) AS attribution_campaign,
+    ANY_VALUE(
+      environment.settings.attribution.content
+      HAVING
+        MIN submission_timestamp
+    ) AS attribution_content,
+    ANY_VALUE(
+      environment.settings.attribution.experiment
+      HAVING
+        MIN submission_timestamp
+    ) AS attribution_experiment,
+    ANY_VALUE(
+      environment.settings.attribution.medium
+      HAVING
+        MIN submission_timestamp
+    ) AS attribution_medium,
+    ANY_VALUE(
+      environment.settings.attribution.source
+      HAVING
+        MIN submission_timestamp
+    ) AS attribution_source,
+    ANY_VALUE(
+      environment.settings.default_search_engine_data.load_path
+      HAVING
+        MIN submission_timestamp
+    ) AS engine_data_load_path,
+    ANY_VALUE(
+      environment.settings.default_search_engine_data.name
+      HAVING
+        MIN submission_timestamp
+    ) AS engine_data_name,
+    ANY_VALUE(
+      environment.settings.default_search_engine_data.origin
+      HAVING
+        MIN submission_timestamp
+    ) AS engine_data_origin,
+    ANY_VALUE(
+      environment.settings.default_search_engine_data.submission_url
+      HAVING
+        MIN submission_timestamp
+    ) AS engine_data_submission_url,
+    ANY_VALUE(environment.system.apple_model_id HAVING MIN submission_timestamp) AS apple_model_id,
+    ANY_VALUE(metadata.geo.city HAVING MIN submission_timestamp) AS city,
+    ANY_VALUE(metadata.geo.db_version HAVING MIN submission_timestamp) AS db_version,
+    ANY_VALUE(metadata.geo.subdivision1 HAVING MIN submission_timestamp) AS subdivision1,
+    ANY_VALUE(normalized_channel HAVING MIN submission_timestamp) AS normalized_channel,
+    ANY_VALUE(normalized_country_code HAVING MIN submission_timestamp) AS country,
+    ANY_VALUE(normalized_os HAVING MIN submission_timestamp) AS normalized_os,
+    ANY_VALUE(normalized_os_version HAVING MIN submission_timestamp) AS normalized_os_version,
+    ANY_VALUE(
+      payload.processes.parent.scalars.startup_profile_selection_reason
+      HAVING
+        MIN submission_timestamp
+    ) AS startup_profile_selection_reason,
+    ANY_VALUE(
+      environment.settings.attribution.dltoken
+      HAVING
+        MIN submission_timestamp
+    ) AS download_token,
```

Comment:
can we rename this to `attribution_dltoken` to fit the pattern with `attribution_source, etc.`?

They're all coming from the same place in the ping (attribution), so keeping their naming convention makes sense.